.github/workflows/ansible-release.yml

```
@@ -0,0 +1,90 @@
+name: ansible-release
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: >-
+          Release Version. Example : 11.1.0
+        required: true
+      ansible-major-version:
+        description: Example 11
+        required: true
+env:
+  CI_COMMIT_MESSAGE: 'Ansible ${{ inputs.ansible-version }}: Dependencies, changelog and porting guide'
+  ANSIBLE_VERSION: '${{ inputs.ansible-version }}'
+
+jobs:
+  build:
+    name: 'Build Ansible (${{ inputs.ansible-version }})'
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+
+    steps:
+      - name: Check out antsibull
+        uses: actions/checkout@v3
+        with:
+          repository: ansible-community/antsibull
+          ref: main
+          path: antsibull
+
+      - name: Pre-create build directory
+        run: mkdir -p antsibull/build
+
+      # This is where the antsibull build-release role expects it by default
+      - name: Check out ansible-build-data under antsibull build directory
+        uses: actions/checkout@v3
+        with:
+          path: antsibull/build/ansible-build-data
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install dependencies
+        working-directory: antsibull
+        run: |
+          python3 -m pip install ansible-core
+          python3 -m pip install antsibull
+          ansible-galaxy collection install community.general
+
+      - name: Checking out to a new branch
+        working-directory: antsibull/build/ansible-build-data
+        run: |
+          git checkout -b "publish-${ANSIBLE_VERSION}"
+
+      - name: Setting the user details
+        working-directory: antsibull/build/ansible-build-data
```

No need to chdir, as mentioned in https://github.com/ansible-community/ansible-build-data/pull/265/files#r1442368714
```suggestion
      - name: Configure user-global Git author and committer metadata
```

---

Updated.